src/components/stop-watch-box/stop-watch-box.tsx

```diff
@@ -1,4 +1,4 @@
-import { Component, State, Prop } from "@stencil/core";
+import { h, Component, State, Prop } from "@stencil/core";
```

Comment:
```suggestion
import { Component, State, Prop } from "@stencil/core";
```

@peterennis we're not using this new import h further in the code.